### ISSUE TITLE
Thank you for your reply

### ISSUE THREAD

**Issue by tianjiaowan** [final-0/Real-ESRGAN#2]
Hello,Dear [final-0]，Your dissertation is professionally written，I have a question, that is, G_loss and D_loss are very different, showing a divergent trend and not converging. Is this normal? Where is the problem? Looking forward to your answer！
![105929EACE3B50D0FB2653847AA5E3C4](https://user-images.githubusercontent.com/105220722/170513254-17a813bc-5cef-4ada-a0a4-5528f474817d.jpg)

**Comment by final-0** [comment]
Yes, these losses are normal. That is because the G_loss contains the VGG loss  (perceptual loss) and GAN loss (adversarial loss), but D_loss has only GAN loss (adversarial loss).
